api/MediaQueryList.json

```diff
@@ -29,10 +29,12 @@
             "version_added": true
           },
           "safari": {
-            "version_added": "5.1"
+            "version_added": "5.1",
+            "notes": "In Safari, <code>MediaQueryList</code> is not based on <code>EventTarget</code>, so you must use <code>addListener()</code> and <code>remoteListener()</code> to observe media query lists. This is expected to change in macOS 11 Big Sur."
```

Comment:
A style note that probably applies in any case: don't introduce the browser if there's no other caveats (e.g., "In Safari" is unnecessary, but "Before Safari 14" would be).

The real issue here is a) whether this bit about macOS 11 is relatively speculative or certain and b) whether it's specific to the version of macOS. If you're not sure it's going to ship, then drop macOS thing entirely, like this:

```suggestion
            "notes": "<code>MediaQueryList</code> is not based on <code>EventTarget</code>, so you must use <code>addListener()</code> and <code>remoteListener()</code> to observe media query lists."
```

If you want the data to be revisited in the future, an issue is the right place for that information.

If you're certain it's going to ship, then I'd be very surprised if it had different behavior based on the version of macOS rather than Safari (e.g., that equivalent Safari versions will have different behavior on macOS 10.15 than macOS 11). There's nothing in the linked bug that suggests that. I'm guessing it's more like this:

```suggestion
            "notes": "Before Safari 14, <code>MediaQueryList</code> is not based on <code>EventTarget</code>, so you must use <code>addListener()</code> and <code>remoteListener()</code> to observe media query lists."
```

If it really is operating-system version dependent, then we can revisit that. But that would be very unusual and I'd want us to document that in this PR for posterity.

Reply:
My intent here is to note that Safari 14, which currently includes this change, is expected to ship with Big Sur. However, you're right that this is extraneous information for our purposes, so I'll remove it.

Historically I've often included the name of the browser the note applies to because it helps in use cases in which a tool is simply displaying all the notes as footnotes. Readability of the note is much improved. However, if our style guide now prefers otherwise, I'll endeavor to change my practices on this.